apps/beeai-sdk/src/beeai_sdk/server/utils.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2025 © BeeAI a Series of LF Projects, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+import asyncio
+from asyncio import CancelledError
+from contextlib import suppress
+
+
+async def cancel_task(task: asyncio.Task[None] | None):
```

Comment:
nitpick: it is strange to accept non in `cancel_task` function.

Reply:
the typical usage pattern is:
```
task = None
try:
    task = asyncio.create_task(...)
finally:
    cancel_task(task)
```
I'd like to avoid the extra `None` check everytime it's used, I'll just rename it to `cancel_task_if_exists`